core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -0,0 +1,571 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+/*
+ * PHASE 1 DONE
+ * PHASE 2 DONE
+ * PHASE 3 DONE
+ * PHASE 4 DONE
+ * PHASE 6 DONE
+ */
+
+/*
+ * EXTENDS ZIO_E_ANY
+ */
+object IO {
+
+  def apply[A](a: => A): Task[A] = effect(a)
+
+  /*
+   * ------------------------------------------------------------------------------------------------
+   * The Following is taken from ZIOFunctions.
+   * ------------------------------------------------------------------------------------------------
+   */
+  // ALL error types in this trait must be a subtype of `UpperE`.
+  // TAKEN FROM ZIO_E_THROWABLE
+  // ALL environment types in this trait must be a supertype of `LowerR`.
+  type IoR = Any
```

Comment:
I'd delete this, I don't think you need it anymore, or all the temporary comments above and below.